pages/etherpad/[[...roomId]].js

```diff
@@ -6,16 +6,16 @@ import { useRouter } from 'next/router';
 
 import { useAuth } from '../../lib/Auth';
 import { useMatrix } from '../../lib/Matrix';
-import EtherpadListEntry from './EtherpadListEntry';
```

Comment:
Now that this file is not being used anymore... can we delete `EtherpadListEntry.js`?